pyproject.toml

```diff
@@ -27,6 +27,7 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Physics",
 ]
 dependencies = [
+  'swxsoc @ git+https://github.com/alrobbertz/swxsoc@integration_fixes',
```

Comment:
NOTE: Temp branch pinned to test changes here before they're merged in https://github.com/swxsoc/swxsoc/pull/12

Reply:
Updated!